tests/lib/config.js

```diff
@@ -358,9 +358,6 @@ describe("Config", () => {
                         browser: true,
                         node: false
                     },
-                    ecmaFeatures: {
-                        globalReturn: false
```

Comment:
I think `ecmaFeatures` is valid as part of the `parserOptions` object, but it's not valid as a top-level property.